.github/workflows/flatpak-test.yml

```diff
@@ -31,4 +31,4 @@ jobs:
       - name: Smoke-test the installed flatpak
         run: |
           . /etc/profile.d/flatpak.sh
-          xvfb-run dbus-run-session containers/flatpak/test-ssh
+          xvfb-run dbus-run-session containers/flatpak/test/test-ssh
```

Comment:
Can you just combine these two changes into a single commit?

Reply:
Done.